backend/src/repository/dalPost.ts

```diff
@@ -58,12 +58,10 @@ export const removeByBoard = async (boardID: string) => {
 
 export const update = async (id: string, post: Partial<PostModel>) => {
   try {
-    const attrUpdate = formatAttributes(post);
```

Comment:
Why're we removing this? It will break on occasions where the post is updated (ie. add a "Needs Attention" tag, move the post around, refresh, and it'll disappear).

Reply:
This and the $set operator were actually causing an error when trying to download posts from the bucket view. Removing the $set didn't cause the error again, and it doesn't break when a post is updated.